cms/templates/group_configurations.html

```diff
@@ -33,31 +31,35 @@ <h1 class="page-header">
         <small class="subtitle">${_("Settings")}</small>
         <span class="sr">&gt; </span>${_("Group Configurations")}
       </h1>
-      <nav class="nav-actions">
-        <h3 class="sr">${_("Page Actions")}</h3>
-        <ul>
-          <li class="nav-item">
-            <a href="#" class="button new-button"><i class="icon-plus"></i> ${_("New Group Configuration")}</a>
-          </li>
-        </ul>
-      </nav>
     </header>
   </div>
 
   <div class="wrapper-content wrapper">
     <section class="content">
       <article class="content-primary" role="main">
-      % if configurations is None:
-        <div class="notice notice-incontext notice-moduledisabled">
-            <p class="copy">
-                ${_("This module is disabled at the moment.")}
-            </p>
+        <div class="wrapper-groups cohort-groups">
+          <h3 class="title">${_("Cohorted Content")}</h3>
+          <p class="copy">${_("A cohorted content group configuration allows different groups of students to view separate content. You can then define what units each cohort can see from the {a_start}Course Outline{a_end}. [Copy TBD]").format(a_start='<a href="{}">'.format(course_outline_url), a_end="</a>")}</p>
+            <div class="ui-loading">
+                <p><span class="spin"><i class="icon icon-refresh"></i></span> <span class="copy">${_("Loading")}</span></p>
+            </div>
         </div>
-      % else:
-        <div class="ui-loading">
-            <p><span class="spin"><i class="icon-refresh"></i></span> <span class="copy">${_("Loading")}</span></p>
+
+        <div class="wrapper-groups experiment-groups">
+          <h3 class="title">${_("Experiment Groups")}</h3>
+          <p class="copy">${_("An experiment group configuration defines how many groups of students are in an experiment. You can then add a content experiment to any unit in the {a_start}Course Outline{a_end}. [Copy TBD]").format(a_start='<a href="{}">'.format(course_outline_url), a_end="</a>")}</p>
+          % if experiment_configurations is None:
+            <div class="notice notice-incontext notice-moduledisabled">
+                <p class="copy">
+                    ${_("This module is disabled at the moment.")}
+                </p>
+            </div>
+          % else:
+            <div class="ui-loading">
+                <p><span class="spin"><i class="icon icon-refresh"></i></span> <span class="copy">${_("Loading")}</span></p>
+            </div>
+          % endif
         </div>
-      % endif
       </article>
       <aside class="content-supplementary" role="complementary">
       <div class="bit">
```

Comment:
Updates to sidebar Help text (lines 66+). NB it should be conditional for 1) Cohorted Content Groups and 2) Experiment Group Configurations. The "Learn More" link should appear at the end of the sidebar text in both cases, each with a different link (to come).

<h3 class="title-3">${_("Cohorted Content Groups")}</h3>

<p>${_("Use cohorted content groups to give groups of students access to specific course content. In addition to course content that is intended for all students, each cohorted content group sees content that you specifically designate as visible to them. By associating one or more cohort groups to a cohorted content group, you can customize the content that a particular cohort group or groups sees in your course.")}</p>

<p>${_("Click {em_start}New cohorted content group{em_end} to add a new cohorted content group. To edit the name of a content group, hover over its box and click {em_start}Edit{em_end}. Cohorted content groups cannot be deleted.").format(em_start='<strong>', em_end="</strong>")}</p>

<h3 class="title-3">${_("Experiment Group Configurations")}</h3>

<p>${_("Use experiment group configurations to how many groups of students are in a content experiment. When you create a content experiment for a course, you select the group configuration to use.")}</p>

 <p>${_("Click {em_start}New Group Configuration{em_end} to add a new configuration. To edit a configuration, hover over its box and click {em_start}Edit{em_end}.").format(em_start='<strong>', em_end="</strong>")}</p>

 <p>${_("You can delete a group configuration only if it is not in use in an experiment. To delete a configuration, hover over its box and click the delete icon.")}</p>